### PR TITLE
improve galleryeditor

### PR DESCRIPTION
**Pull request by juniwbjerde**
image
fixes https://github.com/webkom/lego/issues/1528

### CODE REVIEW

**Comment by erlingfn** [comment]
Nice work! Could you add some before and after pictures to highlight the changes made?

File: app/routes/photos/components/GalleryEditorActions.js [hunk 1]
```diff
@@ -26,37 +26,31 @@ const GalleryEditorActions = ({
...
-            <Icon
-              className={styles.deselectIcon}
-              onClick={onDeselect}
-              name="close"
-            />
-            {selectedCount} Selected
-          </span>
-          <div className={styles.actionBar}>
+      <Flex className={styles.actionsContainer}>
+        <Flex className={styles.actionBar}>
```
Comment (LudvigHz) [inline app/routes/photos/components/GalleryEditorActions.js:30]: Does this do anyting? I.e the inner `Flex` (actionBar) does not have any flex properties. I think you need a `FlexItem` between here, but I'm not 100% sure though.

**Review by LudvigHz — APPROVED** [review]
LGTM
Nice work! This is alot better :smile:

**Comment by LudvigHz** [comment]
> Nice work! Could you add some before and after pictures to highlight the changes made?

@erlingfn You can look at the image in the issue: https://github.com/webkom/lego/issues/1528

File: app/routes/photos/components/GalleryEditorActions.js (hunk 1 above)
Comment (SmithPeder) [reply app/routes/photos/components/GalleryEditorActions.js:30]: +1 The double flex will have an effect, due to the classNames used, but they will override each other.

I recommend that you remove the inner Flex (actionBar),  and delete that css-class, then make some changes to the actionContainer instead

Then make the required changes to `actionContainer` to make it look how you like.

Changing the justify-content to space-between will make it look almoast the same. Maybe change some padding aswell
```sh
justify-content: space-between;
```

**Review by SmithPeder — COMMENTED** [review]
Nice, good job 👍

**Comment by AdrianAndersen** [comment]
Lgtm, good job! :smile: :100:

**Review by AdrianAndersen — APPROVED** [review]

**Pull request merged**